### PR TITLE
8249592 : Robot.mouseMove moves cursor to incorrect location when display scale varies and Java runs in DPI Unaware mode

### PR DESCRIPTION
**Pull request by rajamah**
---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8249592](https://bugs.openjdk.java.net/browse/JDK-8249592): Robot.mouseMove moves cursor to incorrect location when display scale varies and Java runs in DPI Unaware mode

### Reviewers
 * [Sergey Bylokhov](https://openjdk.java.net/census#serb) (@mrserb - **Reviewer**)
 * [Alexey Ivanov](https://openjdk.java.net/census#aivanov) (@aivanov-jdk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/7736/head:pull/7736` \
`$ git checkout pull/7736`

Update a local copy of the PR: \
`$ git checkout pull/7736` \
`$ git pull https://git.openjdk.java.net/jdk pull/7736/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 7736`

View PR using the GUI difftool: \
`$ git pr show -t 7736`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/7736.diff">https://git.openjdk.java.net/jdk/pull/7736.diff</a>

</details>

### CODE REVIEW

File: src/java.desktop/windows/native/libawt/windows/awt_Robot.cpp [hunk 1]
```diff
@@ -28,20 +28,29 @@
...
+
+    int scW = ::GetSystemMetrics(SM_CXVIRTUALSCREEN);
+    int scH = ::GetSystemMetrics(SM_CYVIRTUALSCREEN);
+
+    // The following calculation to deduce mouse coordinates is based on empirical data
+    mouseInput.mi.dx = (x * 65536 + scW - 1) / scW;
+    mouseInput.mi.dy = (y * 65536 + scH - 1) / scH;
+
     ::SendInput(1, &mouseInput, sizeof(mouseInput));
+
```
Comment (mrserb) [inline src/java.desktop/windows/native/libawt/windows/awt_Robot.cpp:53]: This change looks fine, the only suggestion I have is to split the long lines and use the 80 chars per line.

Please also confirm that mach5 is green.

Comment (rajamah) [reply src/java.desktop/windows/native/libawt/windows/awt_Robot.cpp:57]: Mach5 passed all testing with the change. Fixed limit to 80 chars/ line.
thanks

**Review by mrserb — APPROVED** [review]

**Review by aivanov-jdk — APPROVED** [review]

**Pull request closed without merging**